src/main/java/com/google/api/generator/engine/ast/BlockComment.java

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.ast;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class BlockComment implements Comment {
+  public abstract String comment();
```

Comment:
The comment escaper (please see the other comment) can be a field here, or in a separate `escaper` package to be shared between this and `JavaDocComment`.

Reply:
got it, this sounds reasonable to me!